tests/ui/perfherder/alerts-view/status_dropdown_test.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { render, waitFor, cleanup, fireEvent } from '@testing-library/react';
+
+import testAlertSummaries from '../../mock/alert_summaries';
+import testPerformanceTags from '../../mock/performance_tags';
+import repos from '../../mock/repositories';
+import StatusDropdown from '../../../../ui/perfherder/alerts/StatusDropdown';
+
+const testAlertSummary = testAlertSummaries[0];
+const testAlerts = testAlertSummary.alerts;
+const testRepoModel = repos[2];
+
+const testUser = {
+  username: 'mozilla-ldap/test_user@mozilla.com',
+  isLoggedIn: true,
+  isStaff: true,
+  email: 'test_user@mozilla.com',
+};
+
+const testIssueTrackers = [
+  {
+    id: 1,
+    text: 'Bugzilla',
+    issueTrackerUrl: 'https://bugzilla.mozilla.org/show_bug.cgi?id=',
+  },
+  {
+    id: 2,
+    text: 'Github - Servo',
+    issueTrackerUrl: 'https://github.com/servo/servo/pull/',
+  },
+];
+
+const testStatusDropdown = (summaryTags) => {
+  testAlertSummary.performance_tags = summaryTags;
+
+  return render(
+    <StatusDropdown
+      alertSummary={testAlertSummary}
+      user={testUser}
+      updateState={() => {}}
+      repoModel={testRepoModel}
+      updateViewState={() => {}}
+      issueTrackers={testIssueTrackers}
+      bugTemplate={null}
+      filteredAlerts={testAlerts}
+      performanceTags={testPerformanceTags}
+    />,
+  );
+};
+
+afterEach(cleanup);
+
+test("Summary with no tags shows 'Add tags'", async () => {
+  const { getByText } = testStatusDropdown([]);
+
+  const dropdownItem = await waitFor(() => getByText('Add tags'));
+
+  expect(dropdownItem).toBeInTheDocument();
+});
+
+test("Summary with tags shows 'Edit tags'", async () => {
+  const { getByText } = testStatusDropdown(['harness']);
+
+  const dropdownItem = await waitFor(() => getByText('Edit tags'));
+
+  expect(dropdownItem).toBeInTheDocument();
+});
+
+test("Tags modal opens from 'Add tags'", async () => {
+  const { getByText, getByTestId } = testStatusDropdown([]);
+
+  const dropdownItem = await waitFor(() => getByText('Add tags'));
+
+  expect(dropdownItem).toBeInTheDocument();
```

Comment:
We already tested this in "Summary with no tags shows 'Add tags'".
We can safely remove this line, as this functionality is covered by the tests.